rpm/luacheck.spec

```diff
@@ -0,0 +1,41 @@
+Name: luacheck
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: A static analyzer and a linter for Lua
+Group: Development/Tools
+License: MIT
+URL: https://github.com/mpeterv/luacheck
+Source0: luacheck-%{version}.tar.gz
+
+BuildArch: noarch
+
+BuildRequires: tarantool >= 1.7.2.0
+Requires: tarantool >= 1.7.2.0
+
+%description
+Luacheck is a command-line tool for linting and static analysis of Lua code.
+It is able to spot usage of undefined global variables, unused local variables
+and a few other typical problems within Lua programs.
+
+%prep
+%setup -q -n luacheck-%{version}
+
+%install
+mv ./bin/luacheck.lua luacheck.lua
+tarantool install.lua . %{_prefix}/share/tarantool/luacheck
+mkdir luacheck
+cp -r ./src/luacheck/* ./luacheck
+rm -rf src
+mkdir -p %{buildroot}%{_bindir}
+cp ./bin/luacheck %{buildroot}%{_bindir}/
+mkdir -p %{buildroot}%{_prefix}/share/tarantool/luacheck
+cp -R . %{buildroot}%{_prefix}/share/tarantool/luacheck/
+
+%files
+%{_prefix}/share/tarantool/luacheck
+%{_bindir}/luacheck
+
+%changelog
+
+* Tue Jul 10 2018 Ivan Koptelov <ivan.koptelov@tarantool.org> 0.0.1-1
```

Comment:
0.0.0 is Version above, but 0.0.1 here? What is real version should be?

Reply:
Actually we don't use it at all. Packpack set it to VERSION during build or, if it is not possible to get VERSION (e.g 'v0.1) , version is set to 0.0.1
I prefer to set it to the current tag for information needs.

Reply:
So, 0.0.0 above is just boilerplate? Looks weird…

Reply:
We can't remove because it breaks down `rpmbuild`